clang/test/CodeGen/attr-target-version.c

```diff
@@ -659,21 +659,21 @@ int hoo(void) {
 //
 //
 // CHECK: Function Attrs: noinline nounwind optnone
-// CHECK-LABEL: define {{[^@]+}}@fmv_inline._Mfp16fmlMsimd
+// CHECK-LABEL: define {{[^@]+}}@fmv_inline._MsimdMfp16fml
 // CHECK-SAME: () #[[ATTR7]] {
 // CHECK-NEXT:  entry:
 // CHECK-NEXT:    ret i32 14
 //
 //
 // CHECK: Function Attrs: noinline nounwind optnone
-// CHECK-LABEL: define {{[^@]+}}@fmv_inline._Msm4Mfp
```

Comment:
I think it's okay now since FMV as a feature is in beta and the feature is quite new, but I'm curious what the plan is to manage ABI breaks because of changes like this going forward?

Maybe the features should be sorted lexicographically and not according to their priority.

Reply:
Good point, noted. It is worth discussing this again.